org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
+ is defined in Collection and returns a collection of the same class as
the receiver (Set or subclass).

On Wed, Jun 15, 2016 at 10:21 PM, Franco Leonardo Bulgarelli <
notifications@github.com> wrote:

> In org.uqbar.project.wollok.lib/src/wollok.wlk
> <https://github.com/uqbar-project/wollok/pull/744#discussion_r67274524>:
>
> > @@ -407,6 +464,26 @@ package lang {
> >  		override method asSet() = self
> >
> >  		override method anyOne() native
> > +
> > +		/**
> > +		 * Returns a new Set with the elements of both self and another collection.
> > +		 * @returns a Set
> > +		 */
> > +		 method union(another) = self + another
>
> Sorry, where is + defined?
>
> I see union, difference , etc documented as if they return a set, but it
> seems they are not.
>
> method intersection(another) = self.filter({it => another.contains(it)})
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/uqbar-project/wollok/pull/744/files/43910e97fe384ed85998456476fc455b092f32fc#r67274524>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ-EZqSoGm6UCYiyNdTZKDBvu-AllG8jks5qMKUwgaJpZM4I2zBU>
> .
>